assume/common/scenario_loader.py

```diff
@@ -291,10 +291,15 @@ async def load_scenario_folder_async(
     save_frequency_hours = config.get("save_frequency_hours", None)
     sim_id = f"{scenario}_{study_case}"
 
-    bidding_params = config.get("bidding_strategy_params", {})
-    await world.setup(start, end, save_frequency_hours, sim_id, bidding_params, index)
-
-    # add Agents to world
+    bidding_strategy_params = config.get("bidding_strategy_params", {})
+    if "load_learned_path" not in bidding_strategy_params:
```

Comment:
```suggestion
    if "load_learned_path" not in bidding_strategy_params.keys():
```
i would always be explicit here